src/Workload/Microsoft.Maui.Sdk/Sdk/BundledVersions.in.targets

```diff
@@ -13,127 +13,34 @@
-    />
-    <KnownFrameworkReference
-        Condition=" '$(UseMaui)' == 'true' or '$(UseMauiEssentials)' == 'true' "
-        Include="Microsoft.Maui.Essentials"
-        TargetFramework="$(MauiWorkloadTargetFramework)"
-        RuntimeFrameworkName="Microsoft.Maui.Essentials"
-        DefaultRuntimeFrameworkVersion="$(_MauiRuntimePackVersion)"
-        LatestRuntimeFrameworkVersion="$(_MauiRuntimePackVersion)"
-        TargetingPackName="Microsoft.Maui.Essentials.Ref.$(_MauiPlatformName)"
-        TargetingPackVersion="$(_MauiRuntimePackVersion)"
-        RuntimePackNamePatterns="Microsoft.Maui.Essentials.Runtime.**RID**"
-        RuntimePackRuntimeIdentifiers="$(_MauiPlatformName)"
-        Profile="$(TargetPlatformIdentifier)"
-        RuntimePackAlwaysCopyLocal="$(_MauiRuntimePackAlwaysCopyLocal)"
-    />
-  </ItemGroup>
-
   <!-- These implicit <PackageReference/> pull dependencies from NuGet transitively -->
-  <ItemGroup Condition=" '$(UseMaui)' == 'true' or '$(UseMauiCore)' == 'true' ">
-    <PackageReference Include="Microsoft.Extensions.Configuration" Version="@MicrosoftExtensionsServicingPackageVersion@" IsImplicitlyDefined="true">
-      <PrivateAssets Condition=" '$(OutputType)' == 'Library' and '$(AndroidApplication)' != 'true' ">all</PrivateAssets>
-    </PackageReference>
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="@MicrosoftExtensionsPackageVersion@" IsImplicitlyDefined="true">
-      <PrivateAssets Condition=" '$(OutputType)' == 'Library' and '$(AndroidApplication)' != 'true' ">all</PrivateAssets>
-    </PackageReference>
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="@MicrosoftExtensionsPackageVersion@" IsImplicitlyDefined="true">
-      <PrivateAssets Condition=" '$(OutputType)' == 'Library' and '$(AndroidApplication)' != 'true' ">all</PrivateAssets>
-    </PackageReference>
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="@MicrosoftExtensionsServicingPackageVersion@" IsImplicitlyDefined="true">
-      <PrivateAssets Condition=" '$(OutputType)' == 'Library' and '$(AndroidApplication)' != 'true' ">all</PrivateAssets>
-    </PackageReference>
+  <ItemGroup Condition=" '$(UseMauiAssets)' == 'true' ">
+    <PackageReference Include="Microsoft.Maui.Resizetizer" Version="$(MauiVersion)" IsImplicitlyDefined="true" PrivateAssets="all" />
   </ItemGroup>
-  <ItemGroup Condition=" '$(UseMaui)' == 'true' or '$(UseMauiCore)' == 'true' or '$(UseMauiEssentials)' == 'true' ">
-    <PackageReference Include="Microsoft.Maui.Graphics" Version="$(MauiVersion)" IsImplicitlyDefined="true">
+  <ItemGroup Condition=" '$(UseMauiEssentials)' == 'true' ">
+    <PackageReference Include="Microsoft.Maui.Essentials" Version="$(MauiVersion)" IsImplicitlyDefined="true">
       <PrivateAssets Condition=" '$(OutputType)' == 'Library' and '$(AndroidApplication)' != 'true' ">all</PrivateAssets>
     </PackageReference>
```

Comment:
@Redth  Should these still be PrivateAssets=all? Right now, UseMaui in a class library does not add any additional dependencies on the final nupkg, but that may have other issues. If we remove this, then the final nupkg will have all these dependencies.

The downside is that if we are using UseMaui in other repos and are building against preview versions of things that are not public, those versions will now be hardcoded in their final packages. For example, I sometimes use CI builds of maui to build SkiaSharp.Views.Maui and then publish ahead of time. Removing the PrivateAssets=all will now mean I have to wait for things to go public.